src/shape/bounding-box.js

```diff
@@ -346,18 +346,20 @@ class BoundingBox {
     }
 
     /**
-     * Compute the size of the AABB to encapsulate all specified vertices.
+     * Compute the min and max bounding values to encapsulate all specified vertices.
      *
      * @param {number[]|Float32Array} vertices - The vertices used to compute the new size for the
      * AABB.
+     * @param {Vec3} min - Computed min value.
+     * @param {Vec3} max - Computed max value.
      * @param {number} [numVerts] - Number of vertices to use from the beginning of vertices array.
      * All vertices are used if not specified.
      */
-    compute(vertices, numVerts) {
-        numVerts = numVerts === undefined ? vertices.length / 3 : numVerts;
+    static computeMinMax(vertices, min = new Vec3(), max = new Vec3(), numVerts = vertices.length / 3) {
```

Comment:
Why does this create new vecs if none are supplied. They don't get returned so it seems a bit pointless?

Reply:
good point. fixing, thanks!